### PR TITLE
Beta v6.20.1 => v6.20.2

### PR DESCRIPTION
**Pull request by MichaIng**
#### Changes / Improvements / Optimisations
- DietPi-Config | RPi: Added info to "hifiberry-dacplus" sound device overlay that it supports the AMP2 DAC. Thanks to @Lat Dior for reporting the issue and testing the solution: https://dietpi.com/phpbb/viewtopic.php?t=5396
- DietPi-Software | WireGuard VPN is now available for installation. Is can be easily configured as client or as server, passing either all client traffic through the tunnel, local network access only, or server access only, e.g. as well in combination with Pi-hole, to have remote ad blocking.
- DietPi-RAMlog/Logclear | The internally rotated LetsEncrypt (CertBot) log files are now removed, hourly with DietPi-RAMlog enabled, or when executing DietPi-Logclear manually.

#### Bug Fixes
- DietPi-Software | Deluge: Resolved a visual-only error in systemd unit, since same-line comments are not allowed anymore. Thanks to @Gabba for reporting this issue: https://dietpi.com/phpbb/viewtopic.php?t=5378
- DietPi-Software | No-IP: Fixed an issue, where on "Generic device" (ID 22), the wrong binary could have been installed. As well added failsafe "/usr/local/[bin|etc]" directory pre-creation, in case not present on the system or manually removed by user. Thanks to @walker93 for reporting this issue: https://github.com/Fourdee/DietPi/issues/2426
- DietPi-Config | RPi: gpu_mem_256/512 are now capped when being applied (to prevent value exceeding limits of each board type): https://github.com/Fourdee/DietPi/issues/2417
- DietPi-Config | Resolved a visual-only issue, where WiFi interface starts throw an error message about unsupported power management: https://github.com/Fourdee/DietPi/issues/2198
- DietPi-WiFi-Monitor | Resolved an issue, where DietPi-WiFi-Monitor stuck in a reconnection loop, if no valid WiFi interface was detected or the WiFi adapter is unplugged.
- DietPi-Cloudshell | Resolved an issue where CPU scene failed on VM and other devices that do not support CPU governors: https://github.com/Fourdee/DietPi/issues/2428
- General | Pinebook: Resolved issue with failed apt-get upgrades due to missing armbian-release data file. Also switched to stable kernel packages and kernel updates over APT are now enabled.

### CODE REVIEW

File: dietpi/dietpi-cleaner [hunk 1]
```diff
@@ -24,9 +24,12 @@
 	G_INIT
 	#Import DietPi-Globals ---------------------------------------------------------------
 
+	#Exit trap (dietpi-globals)
+	G_EXIT_CUSTOM(){ unset aEnabledCleaners; }
```
Comment (MichaIng) [inline dietpi/dietpi-cleaner]: @Fourdee 
However, I am still not sure if this has any effect or all allocated RAM usage is anyway freed, when the script exits.

https://unix.stackexchange.com/a/360377
- The TO mixes things up there, but the answer states as well:
> In any case, the variables themselves will be disposed of when the script ends and the operating system reclaims the shell's memory.
> ...
> There is no need to blank or unset variables at the end of a shell script in any way.

And a second one: https://stackoverflow.com/a/12998828

Comment (Fourdee) [reply dietpi/dietpi-cleaner]: @MichaIng 

Interesting, good find 👍 
So we only need to delete vars during runtime (if required).

Comment (MichaIng) [reply dietpi/dietpi-cleaner]: @Fourdee 
Jep seems like. I mean also even large arrays take a tiny amount of RAM, compared to the whole shell, but especially in large scripts where variables might be used doubled (e.g. dietpi-software/config) I would consider it still good practice to unset any used variable, after used.

**Comment by Fourdee** [comment]
Ok, will push this today, just want to run some local testing 1st.

**Comment by Fourdee** [comment]
@MichaIng 

> Also it forcefully adds the comment, if not setting or comment is presen

👍 
Not ideal, but its fine, no harm in adding the entry as commented out.

I also like the visual output of ```G_CONFIG_INJECT```, shows whats being done for the end users that want to see it.

**Pull request merged**